devtools/dump_devinfo.py

```diff
@@ -464,24 +728,24 @@ async def get_smart_test_calls(device: SmartDevice):
 
     if "child_device" in components:
         child_components = await _make_requests_or_exit(
-            device,
-            [SmartRequest.get_child_device_component_list()],
+            protocol,
+            SmartRequest.get_child_device_component_list().to_dict(),
             "child device component list",
             batch_size=1,
             child_device_id="",
         )
         successes.append(
             SmartCall(
                 module="child_component_list",
-                request=SmartRequest.get_child_device_component_list(),
+                request=SmartRequest.get_child_device_component_list().to_dict(),
```

Comment:
Any reason not to do the serialization (i.e., `to_dict()` call) inside wherever this is used?